.eslintrc.js

```diff
@@ -24,11 +24,17 @@
       "argsIgnorePattern": "(^reject$|^_$)",
       "varsIgnorePattern": "(^_$)"
     }],
+    "brace-style": [2, "1tbs", {
+      "allowSingleLine": true
+    }],
     "quotes": [2, "single"],
     "require-jsdoc": 0,
     "valid-jsdoc": 0,
     "comma-dangle": 0,
     "prefer-const": 2,
+    // browser/node support for rest params isn't ready yet
```

Comment:
sg. 
its node 5 without a flag that's the problem for us.

we haven't officially retired support but we could soon.